frontend/src/pages/Help/PDFViewer/PDFViewer.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Document } from 'react-pdf';
+
+import Button from 'components/Button';
+import { FIRST_PAGE_INDEX, LAST_PAGE_INDEX } from 'constants/pdfViewer';
+
+import { ButtonBox, DocumentBox, PageNumberText, PDFWrapper, StyledPage } from './styled';
+
+interface IPDFViewer {
+  src: string;
+  pageNumber: number;
+  previousPage: () => void;
+  nextPage: () => void;
+  onDocumentLoadSuccess: (pdf: any) => void;
```

Comment:
Could you please try to find the type for `pdf` prop?

Reply:
Unfortunately, I couldn't find a proper type for pdf prop.  Any other type throws an error